File: modul6/C/PRAK603-2310817210023-RifkyPutraMahardika.c
#include <stdio.h>

int main() {
    int i, j, n1, n2;
    int hasil[50];

    printf("Input batas angka tiap baris:\n");
    scanf("%d %d", &n1, &n2);
    int angka[n1][n2];

    if (n1 == n2) {
        printf("Masukkan angka :\n");
        for (i = 0; i < 2; i++) {
            for (j = 0; j < n2; j++) {
                scanf("%d", &angka[i][j]);
                hasil[j] = angka[0][j]*angka[1][j];
            }
        }

        printf("Output :\n");
        for (j = 0; j < n2; j++) {
            printf("%d ", hasil[j]);
        }
    } else {
        printf("Output :\nJumlah tidak sama\n");
    }

return 0;
}

File: modul6/Python/PRAK603-2310817210023-RifkyPutraMahardika.py
n1, n2 = map(int, input("Input batas angka tiap baris:\n").split())

if n1 == n2:
    print("Masukkan angka :")
    angka = [list(map(int, input().split())) for i in range(2)]
    hasil = [angka[0][j] * angka[1][j] for j in range(n2)]

    print("Output:")
    print(*hasil)

elif n1 != n2:
    print("Output:")
    print("Jumlah tidak sama")